unitylibs/core/workflow/workflow-upload/action-binder.js

```diff
@@ -312,6 +313,7 @@ export default class ActionBinder {
         el.addEventListener('dragover', this.preventDefault);
         el.addEventListener('dragenter', this.preventDefault);
         el.addEventListener('drop', async (e) => {
+          sendAnalyticsEvent(new CustomEvent('Drag and drop|UnityWidget'));
```

Comment:
nit: we are using camelcase for event name elsewhere
```suggestion
          sendAnalyticsEvent(new CustomEvent('dragAndDrop|UnityWidget'));
```